fsrefstore.go

```diff
@@ -216,6 +216,7 @@ func (f *FileManager) readURLDataObj(c cid.Cid, d *pb.DataObj) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	req = req.WithContext(ctx)
 
 	req.Header.Add("Range", fmt.Sprintf("bytes=%d-%d", d.GetOffset(), d.GetOffset()+d.GetSize_()-1))
```

Comment:
Have we been doing these changes in other PRs? This may make sense, but I don't think we're really doing any aborting based on context cancellation elsewhere.

Reply:
This was more for completeness, I don't feel strongly about it.  I can remove if you prefer.

Reply:
I suspect it's safe to drop this for now and we can put it in later (i.e. make this PR a bubble-up only/no-code PR).